nsqd/http.go

```diff
@@ -24,6 +24,13 @@ import (
 	"github.com/nsqio/nsq/internal/version"
 )
 
+var binaryParams = map[string]bool{
```

Comment:
I called this "boolParams" because it could be used for other boolean params besides "binary" (we don't really have any other simple boolean params right now though).

If it was really only for "binary", we wouldn't put it in the file/package scope, we'd just put it in the only function that would use it.